platform/node/src/node_map.cpp

```diff
@@ -474,7 +474,7 @@ void NodeMap::startRender(NodeMap::RenderOptions options) {
 
     map->renderStill(camera, options.debugOptions, [this](const std::exception_ptr eptr) {
```

Comment:
`StillImageCallback` passes `std::exception_ptr` by value, const can be removed.

Reply:
This will come on another PR fixing other warnings.